frontend/.eslintrc

```diff
@@ -17,7 +17,8 @@
     "react/prop-types": 0,
     "no-underscore-dangle": 0,
     "import/imports-first": ["error", "absolute-first"],
-    "import/newline-after-import": "error"
+    "import/newline-after-import": "error",
+    "max-len": 0,
```

Comment:
I think we want to continue enforcing line length, so probably want to remove this.